Thai/Travel/Travel.md

```diff
@@ -0,0 +1,7 @@
+﻿You will fall in love with Thailand. It is amazingly beautiful. We have plenty of stunning beaches and crystal clear water. Imagine you are at the sea now, what would you like to do?
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.